src/WidgetBase.ts

```diff
@@ -496,6 +498,14 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> implement
 
 		return dNode;
 	}
+
+	private _runAfterConstructors(): void {
+		const afterConstructors = this.getDecorator('constructor');
```

Comment:
should we call this `afterConstructor`?

Reply:
I aligned it with `beforeRenders`, `afterRenders`. it returns an array after all.

Reply:
I meant the decorator name, `this.getDecorator('constructor')` -> `this.getDecorator('afterConstructor')`